src/configuration/app/build.md

```diff
@@ -0,0 +1,135 @@
+# Build and deploy
+
+The `.platform.app.yaml` file provides a number of ways to control how an application gets turned from a directory in Git into a running application.  There are three blocks that control different parts of the process: the build flavor, dependencies, and hooks.  The build process will run the build flavor, then install dependencies, then run the user-provided build hook.  The deploy process will run the deploy hook.
+
+## Build
+
+The `build` defines what happens when building the application.  Its only property is `flavor`, which specifies a default set of build tasks to run. Flavors are language-specific.
+
+### PHP (`composer` by default)
+
+`composer` will run `composer --no-ansi --no-interaction install --no-progress --prefer-dist --optimize-autoloader` if a `composer.json` file is detected.
```

Comment:
Maybe run `platform build` (`--clone`) on the local environment, to stay consistent?